### PR TITLE
📝 Add update env reminder

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## What do these changes do?

These changes introduce a reminder and a link to the documentation about updating ENV variables in `repo.config`s

## Related issue/s

* https://github.com/ITISFoundation/osparc-ops-environments/issues/404
* https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/136

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
cool thank you! let me know when this needs to go in

**Review by GitHK — APPROVED** [review]
👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ or from https://gitmoji.dev/
 
 <!-- Give REVIEWERS some hits or code snippets on how could this be tested -->
 
+## Dev Checklist
+
+- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))
```
Comment (pcrespov) [inline .github/PULL_REQUEST_TEMPLATE.md:46]: Why not just putting it in the existing devops checklist?

**Review by pcrespov — APPROVED** [review]

**Review by mrnicegyu11 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (YuryHrytsuk) [reply .github/PULL_REQUEST_TEMPLATE.md:46]: lte's try it this way and then see

**Pull request merged**